src/MessagingAttachment/MessagingAttachment.tsx

```diff
@@ -34,7 +34,9 @@ export const MessagingAttachment: React.FC<Props> = ({
   uploadComplete,
 }: Props) => {
   return (
-    <FlexBox className={cssClass("ParentContainer")}>
+    <FlexBox
+      className={cx(cssClass("ParentContainer"), isUpload && cssClass("ParentContainer--IsUpload"))}
```

Comment:
Any reason why we're adding a new `ParentContainer--IsUpload`  class here? Seems like the only CSS selector for this class is also being applied to `ParentContainer`

Reply:
The only style being added here is added to `.MessagingAttachment--ParentContainer.MessagingAttachment--ParentContainer--IsUpload`, a more specific selector than `.MessagingAttachment--ParentContainer`. I add the `overflow: visible` property to this selector, which finds "uploaded" attachments (aka attachments in drafts in the input box), because we want to be able to see the x icon that flows out of the edge of the box. I need it to be more specific than `.MessagingAttachment--ParentContainer` because that has a style of `overflow:hidden` which helps most of our attachments not run off the edge of the page.

Reply:
ah my bad, thought we were applying them separately, just realized it's chained